frontend/src/modules/organization/pages/organization-common-page.vue

```diff
@@ -67,35 +83,46 @@ const getOrganizations = (body?: any) => {
     ...savedBody.value,
     ...body,
   };
-  OrganizationService.query({
+  loading.value = true;
+  OrganizationService.organizationsList({
     ...savedBody.value,
     orderBy: sorting.value,
+    excludeSegments: true,
   })
     .then((data: Pagination<Organization>) => {
       organizations.value = data.rows;
-      totalOrganizations.value = data.count;
+      pagination.value.total = data.count;
     })
     .catch((err: Error) => {
       organizations.value = [];
-      totalOrganizations.value = 0;
+      pagination.value.total = 0;
     })
     .finally(() => {
       loading.value = false;
     });
 };
 
+const getOrganizationsCount = () => {
+  OrganizationService.organizationsList({
+    excludeSegments: true,
+    limit: 1,
+  })
+    .then((data: Pagination<Organization>) => {
+      totalOrganizations.value = data.count;
+    })
+    .catch((err: Error) => {
+      totalOrganizations.value = 0;
+    });
+};
```

Comment:
If this endpoint is not optimized to only get the count, shouldn't we simply use the count already returned when you get the list?